views/users/profile.hbs

```diff
@@ -25,27 +25,52 @@
   </section>
 
   <section class="side side-right">
-    <a href="#"><img src="Images/add.png" width="50px" disabled/></a>
+    <form action="/newtab" method="POST" >
+      <div class="form-div">
+        <select name="category" class="form-input" height="10">
+          <option value=" ">Pick a category</option>
+          <option name="video" value="video">Video</option>
+          <option name="article" value="article">Article</option>
+          <option name="podcast" value="podcast">Podcast or Audio</option>
+          <option name="book" value="book">Book</option>
+          <option name="fullsite" value="fullsite">Web site</option>
+        </select>
+      </div>
+      <div class="form-div">
+        <input type="text" name="description" class="form-input" placeholder="Add the link"></input>
```

Comment:
`<input />` is a self closing element so remove the `</input>` and add the slash at the opening one.
Second; is this a one line element? if yes, than it's okay, but as the name is `description` I would assume multiline which would make it a `<textarea></textarea>`.

So change the name if it's a one line